ab_tests/ab_tests.yaml

```diff
@@ -4,3 +4,7 @@
 - Example:
   - A
   - B
+- UxmPracticeTest:
```

Comment:
The thing I always need to check by seeing it running, is that this test name corresponds with the expected header in the app. I'm always 99% sure, but never quite 100%. 😰 
(It looks good to me FWIW, and checks out in the regenerated specs)